### DISCUSSION TITLE
Please support short time notation 12h34 (withoun 'm' suffix)

### DISCUSSION THREAD

**Discussion by QuAzI** [jotaen/klog#316] [Feature ideas & discussions]
Please support `12h34` format the same way as `12h34m` as only minutes possible after hours

**Comment by jotaen** [comment]
Thanks for suggesting this. I agree that `12h34` should be non-ambiguous in practice – in theory, it could also be interpreted as `12h34s`, though I’d consider this rather counterintuitive.

The main reasoning for using `12h34m` is that it’s more expressive and explicit. I also see value in keeping the overall number of syntax variations as small as possible. This is a bit of a balancing act, though, and I think personal preferences might diverge here.

**Reply by QuAzI** [reply]
As I know, there is no way to track seconds in klog? Also, the same short time format used in some other time tracking software
